test/Main.purs

```diff
@@ -0,0 +1,11 @@
+module Test.Main where
+
+import Prelude
+
+import Effect (Effect)
+import Effect.Class.Console (log)
+
+main :: Effect Unit
+main = do
+  log "🍝"
+  log "You should add some tests."
```

Comment:
I'm aware this test does nothing at the moment, but we may (in the future) want to write something small here which minimally exercises the bindings as a sanity check when making future changes. Having this directory ready to go and running in CI makes it easier for contributors to help add tests in the future.

Reply:
I think this is the right call. Having the test structure in place will definitely make for a smoother experience if someone wants to add some tests.